src/ResourceManager/ResourceGraph/Commands.ResourceGraph/Az.ResourceGraph.psd1

```diff
@@ -0,0 +1,131 @@
+#
+# Module manifest for module 'AzureRM.ResourceGraph'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 9/6/2018
+#
+
+@{
+
+        # Script module or binary module file associated with this manifest.
+        # RootModule = ''
+
+        # Version number of this module.
+        ModuleVersion = '0.1.0'
+
+        # Supported PSEditions
+        CompatiblePSEditions = 'Core'
```

Comment:
@demyanenko nit: this should be

`CompatiblePSEditions = 'Core', 'Desktop'`

(not a blocker, can be done at a later time)